openstreetmap-carto.lua

```diff
@@ -0,0 +1,447 @@
+-- For documentation of Lua tag transformations, see:
+-- https://github.com/openstreetmap/osm2pgsql/blob/master/docs/lua.md
+
+-- Custom keys that are defined by this file
+custom_keys = { 'z_order', 'osmcarto_z_order' }
+
+-- Objects with any of the following keys will be treated as polygon
+polygon_keys = { 'building', 'landuse', 'amenity', 'harbour', 'historic', 'leisure',
+      'man_made', 'military', 'natural', 'office', 'place', 'power',
+      'public_transport', 'shop', 'sport', 'tourism', 'waterway',
+      'wetland', 'water', 'aeroway', 'abandoned:aeroway', 'abandoned:amenity',
+      'abandoned:building', 'abandoned:landuse', 'abandoned:power', 'area:highway' }
+
+-- Objects with any of the following key/value combinations will be treated as polygon
+polygon_values = {
+      {'highway', 'services'},
```

Comment:
Adding Idea from @jojo4u in https://github.com/gravitystorm/openstreetmap-carto/issues/137#issuecomment-124904987 
[rest_area](http://wiki.openstreetmap.org/wiki/Tag:highway%3Drest_area) should be added. 
We have 6342 ways with this tag and about 1/3 have area=yes on it.
This is currently [rendered](https://github.com/gravitystorm/openstreetmap-carto/blob/0c70682a69c91b833399c1dd74409eee79ce4af1/amenity-points.mss#L2026).